pylib/anki/browser.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright: Ankitects Pty Ltd and contributors
+# License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+
+class BrowserConfig:
+    ACTIVE_CARD_COLUMNS_KEY = "activeCols"
+    ACTIVE_NOTE_COLUMNS_KEY = "activeNoteCols"
+    CARDS_SORT_COLUMN_KEY = "sortType"
+    NOTES_SORT_COLUMN_KEY = "noteSortType"
+    CARDS_SORT_BACKWARDS_KEY = "sortBackwards"
+    NOTES_SORT_BACKWARDS_KEY = "browserNoteSortBackwards"
+    CARDS_GEOMETRY_KEY_PREFIX = "editor"
+    NOTES_GEOMETRY_KEY_PREFIX = "editorNotesMode"
```

Comment:
The geom keys are Qt specific and unlikely to be accessed elsewhere, so you could store them directly on CardState/NoteState ... but not a big deal either way.